templates/store/snap-details.html

```diff
@@ -103,18 +103,18 @@ <h1 class="p-heading--two p-snap-heading__name">{{ snap_title }}</h1>
     {% include "store/snap-details/_channel_map.html" %}
   {% endif %}
 
-  {% if screenshots or videos %}
+  {% if screenshots %}
     <div class="p-strip is-shallow">
       <div class="row">
         <div class="p-carousel" id="js-snap-screenshots">
           <div class="swiper-container">
             <div class="swiper-wrapper">
-              {% for video in videos %}
-                <div class="p-carousel__item--video swiper-slide">
-                  <div class="js-video-slide" data-video-type="{{video.type}}" data-video-url="{{video.url}}">
-                  </div>
-                </div>
-              {% endfor %}
+              <!--{% for video in videos %}-->
```

Comment:
Can we comment it out from template `{# .... #}` rather then still rendering it into HTML?